core/src/main/java/org/mule/runtime/core/util/IOUtils.java

```diff
@@ -262,4 +272,60 @@ public static long copyLarge(Reader input, Writer output) throws IOException
         }
         return count;
     }
+
+    /**
+     * Transforms an Object attachment into a DataHandler of its corresponding type.
```

Comment:
jdoc should be agnostic of the particular use you're planning to do of this method. It should only refer to data handlers